modules/swagger-codegen/src/main/resources/csharp/ApiResponse.mustache

```diff
@@ -33,7 +33,7 @@ namespace {{packageName}}.Client
         /// <param name="statusCode">HTTP status code.</param>
         /// <param name="headers">HTTP headers.</param>
         /// <param name="data">Data (parsed HTTP body)</param>
-        public ApiResponse(int statusCode, IDictionary<string, string> headers, T data)
+        public ApiResponse(int statusCode, IDictionary<string, string[]> headers, T data)
```

Comment:
NOTE: Because this generator isn't currently intended to be regenerated over existing code, I wouldn't consider this a breaking change.

Reply:
n/m. I was looking at this wrong.  This *is* a breaking change.

This should probably be an overloaded constructor which converts the values to a comma separated value per [RFC 2616](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2)

Reply:
Actually, from the POV of the developer of the consuming application I would prefer this to be a breaking change.
Considering that the previous value field didn't contains any real values at all I don't think there will be many broken builds, because one will notice the wrong values immediately and cannot use it as it is before this fix.
If someone get's a broken build, than he or she will see that there can be multiple values and can handle these as they need.
Combining multiple values back into one delimited string only to have the consumer splitt it up again does not seem to be efficient. If the consumer needs the values as a single combined string, than I would leave that up to the consumer to do so.

Reply:
I just fixed it. It's no longer a breaking change for generated netframework clients.

It is a "breaking" (didn't work previously) change for netstandard clients, but they never received the correct values, so I think this is acceptable.